README.md

```diff
@@ -286,6 +287,9 @@ spec:
 
 Can be found at: `mellanox.com_v1alpha1_hostdevicenetwork_cr.yaml`
 
+## Pod Security Policy
+Network-operator supports [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/). When deployed with `psp.enabled=True`, privileged PSP is created and applied to all network-operator's pods. Requires [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in) to be enabled.
```

Comment:
as this README is not related to Helm, small nit:

> When deployed with `psp.enabled=True`

When NicClusterPolicy is created with  `psp.enabled=True`